src/qt/locale/bitcoin_zh_TW.ts

```diff
@@ -2577,8 +2577,8 @@ Note:  Since the fee is calculated on a per-byte basis, a fee of "100 satoshis p
         <translation><numerusform>预计在等待 %n 个区块后会有第一个确认。</numerusform></translation>
     </message>
     <message>
-        <source>Warning: Invalid Bitcoin address</source>
-        <translation>警告: 比特幣地址無效</translation>
+        <source>Warning: Invalid Dogecoin address</source>
+        <translation>警告: 狗狗幣地址無效</translation>
```

Comment:
Retouch the sentence.
```suggestion
        <translation>警告：無效的狗狗幣地址</translation>
```